### DISCUSSION TITLE
JFA incorrectly categorizes enabled users as disabled.

### DISCUSSION THREAD

**Discussion by gomaaz** [hrfee/jfa-go#370] [Q&A] [answered]
Hi,

I recently noticed that there's no button to explicitly disable enabled users.
In the screenshot, you can see that a user is marked as active, but the buttons above don’t change accordingly.
I also tried setting up a fresh Docker container with a new configuration file, but I’m seeing the same result.
Has anyone else encountered a similar issue?

Thanks in advance! By the way, great piece of software!

Emby Beta: 4.9.0.30
Jfa-Go: unstable

![image](https://github.com/user-attachments/assets/337daf5f-4446-43fc-9fdf-135786979f64)

**Comment by xawoger** [comment]
In my instance it does show up. The button to disable is there.

**Reply by gomaaz** [reply]
are you running jfago unstable and which version of emby are you running?

**Reply by xawoger** [reply]
I am using it with Jellyfin, and I am running stable/latest branch of jfago.

**Reply by gomaaz** [reply]
I will test it with a fresh install of latest.

**Comment by hrfee** [comment, marked as answer]
Sorry, this was a regression since I adjusted the account page appearance; I derive account-disabled-ness from a class on the disabled "badge", which I had changed. This sillyness still occurs but now for the correct class "hidden" in bb41bc3.

**Reply by gomaaz** [reply]
thanks for the reply! And many thanks for your great software!

**Reply by gomaaz** [reply]
The button disable reappeared! :)
